pkg/storage/storageclass.go

```diff
@@ -79,9 +83,7 @@ func (builder *ClassBuilder) WithReclaimPolicy(
 		glog.V(100).Infof("The reclaimPolicy of the storageclass is empty")
 
 		builder.errorMsg = "storageclass 'reclaimPolicy' cannot be empty"
-	}
 
-	if builder.errorMsg != "" {
```

Comment:
If you remove this condition we add invalid reclaimPolicy to builder. Please revert it back

Reply:
The diff here on Github makes it look like that would happen.  It makes it look confusing when its not.

![image](https://github.com/user-attachments/assets/a631f598-27dd-4623-aa74-dc5261aa7b6d)